src/Booking.js

```diff
@@ -0,0 +1,11 @@
+class Booking {
```

Comment:
I am guessing your Booking and Room classes are mainly a place to cleanup/organize the fetched data?